test/integration/scenarios/faultTests/README.md

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+SPDX-License-Identifier: MIT
+-->
+# Overview
+
+This is a framework for symphony fault tests. 
+
+# How to add a new fault test
+1. Add failpoints in codes
+
+Add a comment line like example blow. Gofail package will translate it to the failpoint code when compiled with failpoint enabled.
+```
+// gofail: var beforeProviders string
+```
+
+2. Add a new fault test case
+
+There are already two fault tests - solution upgrade and workflow materialize in the faultTests folder. You can add a new case in the [constants.go](./constants.go) you want to use existing fault tests with new failpoint.
```

Comment:
Where is the constants.go? Would be better if you can summarize existing failure point to showcase the coverage.

Reply:
fixed. the file was moved after refactor.